checkov/terraform/graph_builder/variable_rendering/safe_eval_functions.py

```diff
@@ -265,12 +265,18 @@ def terraform_try(*args: Any) -> Any:
         "try evaluates all of its argument expressions in turn and returns the result of the first one that does not
         produce any errors."
     """
+    accepted_exception = "'NoneType' object is not subscriptable"
```

Comment:
could you explain why is this considered an "accepted" exception? Perhaps had a comment about it in the code as well?

Reply:
Added comment :)

Reply:
<p></p>